Resources/Prototypes/Entities/Constructible/Ground/gascanisters.yml

```diff
@@ -35,6 +35,10 @@
     - type: GasCanisterVisualizer
       sprite: Constructible/Atmos/canister.rsi
       stateConnected: can-connector
+      stateO0: can-o0
```

Comment:
Instead of making 4 layers and only showing one at a time, just make a single one and change its icon state?

Also maybe make this `shader: unshaded` for fancy lights.

Reply:
Done.